server/mergin/sync/schemas.py

```diff
@@ -362,6 +362,7 @@ class ProjectAccessDetailSchema(Schema):
     username = fields.String()
     name = fields.String()
     project_permission = fields.String()
+    project_role = fields.String()
```

Comment:
Is this ProjectAccessDetail schema used in public apis? If not I suggest to rename fields to 'workspace_role', 'project_role' and 'role' (combined)

Reply:
see addressing comments commit